.vscode/extensions.json

```diff
@@ -5,7 +5,8 @@
 	// List of extensions which should be recommended for users of this workspace.
 	"recommendations": [
 		"bierner.lit-html",
```

Comment:
Is the vscode workspace files needed in commit

Reply:
Removed